apps/mull-ui/src/app/components/chat-bubble-list/chat-bubble-list.tsx

```diff
@@ -0,0 +1,30 @@
+import { Post } from 'apps/mull-ui/src/generated/graphql';
+import { avatarUrl } from 'apps/mull-ui/src/utilities';
+import React from 'react';
+import ChatBubble from '../../components/chat-bubble/chat-bubble';
+
+const formatDate = (date: Date) => {
+  return `${date.getHours()}:${String(date.getMinutes()).padStart(
+    2,
+    '0'
+  )} ${date.toLocaleDateString()}`;
+};
+
+export const chatBubbleList = ({ data }) => {
```

Comment:
after implementing the interface above you need the following
```suggestion
export const chatBubbleList = ({ posts}: ChatBubbleListProps) => {
```